### DISCUSSION TITLE
Children selector

### DISCUSSION THREAD

**Discussion by melloc01** [ben-rogerson/twin.macro#266] [General] [answered]
Is there any plan to integrate https://github.com/benface/tailwindcss-children to the core features?

**Comment by ben-rogerson** [comment, marked as answer]
~~Sure is - [that plugin uses `addVariant`](https://github.com/benface/tailwindcss-children/blob/master/index.js#L19) which is next on the list to gain support~~.

Edit: After taking a better look into what's involved here, I can say that support for `addVariant` isn't possible in twin.

**Reply by ben-rogerson** [reply]
addVariant support has already been added, only took a rewrite to achieve :D
